www/src/pages/components/forms.js

```diff
@@ -410,11 +410,21 @@ export default withLayout(function FormControlsSection({ data }) {
       <p>
         Bootstrap scopes the <code>:valid</code> and <code>:invalid</code>{' '}
         styles to parent <code>.was-validated</code> class, usually applied to
-        the <code>Form</code> (you can use the <code>validated</code> prop as a
-        shortcut). Otherwise, any required field without a value shows up as
-        invalid on page load. This way, you may choose when to activate them
+        the <code>{'<Form>'}</code> (you can use the <code>validated</code> prop
+        as a shortcut). Otherwise, any required field without a value shows up
+        as invalid on page load. This way, you may choose when to activate them
         (typically after form submission is attempted).
       </p>
+      <p>
+        For custom Bootstrap form validation messages, you’ll need to add the{' '}
+        <code>noValidate</code> boolean attribute to your{' '}
+        <code>{'<Form>'}</code>. This disables the browser default feedback
```

Comment:
don't we set noValidate to false automatically in the component? Should we?

Reply:
Do you mean noValidate = true?  It's currently not set by default.  I think it's better to follow the default behavior here by having noValidate as an opt in.